tracking: add `min_duration` argument to `lk.filter_tracks` (#587)
lumicks/pylake: changelog.md

* Added more options for plotting color channels for images:
  * Shortcuts `"r"`, `"g"`, and `"b"` can now be used for plotting single color channels in addition to `"red"`, `"green"`, and `"blue"`.
  * Two-channel visualizations can be plotted using `"rg"`, `"gb"`, or `"rb"`.

> * Added `duration` property to `KymoTrack` which returns the duration (in seconds) that the track was observed.
> * Added option to filter tracks with `lk.filter_tracks(tracks, minimum_duration=duration_in_seconds)` by track duration (in seconds).
> * Added `KymoTrackGroup.filter()` to filter tracks by duration or length (number of identified track points) in-place.

Reviewer: ```suggestion
* Added `duration` property to `KymoTrack` which returns the track duration in seconds.
* Added option to filter tracks by duration in seconds using `lk.filter_tracks(tracks, minimum_duration=duration_in_seconds)`.
* Added `KymoTrackGroup.filter()` to filter tracks in-place. `tracks.filter(minimum_duration=2)` is equivalent to `lk.filter_tracks(tracks, minimum_duration=2)`.
```
